From 0644ca6e7afdd18156165c08048c444e40851b03 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 14 Mar 2017 20:05:53 +0000
Subject: [PATCH 0830/1077] MANTIS 0044501 Merged revision(s) 1171 from trunk:
 MANTIS 0044501 change: mSysProcessTerminate::setShutdownHook does not work as
 expected

massai_systools.lib
V1.0.0.36

- added new function setShutdownHook with window title and block shutdown reason for shutdown notification on windows 7
- creates an invisible window to block shutdown


........


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1172 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/cpp/Systools/makefile.mak              | 2 +-
 .../Massai/cpp/Systools/src/TestSysProcessTerminate.cpp         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/Systools/makefile.mak b/SelfServiceCommon/Massai/cpp/Systools/makefile.mak
index 49107a4f..f76d0b7f 100644
--- a/SelfServiceCommon/Massai/cpp/Systools/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/Systools/makefile.mak
@@ -193,7 +193,7 @@ MY_BIN14_OBJS =         $(_OBJ)\TestSysServiceMonitor.obj
 
 MY_BIN15 =              $(_BIN)\TestSysProcessTerminate.exe
 
-MY_BIN15_USER_LIBS =    $(LIB_SYSTOOLS) $(LIB_TOOLS) $(LIB_TEXT)  $(LIB_TRACE)
+MY_BIN15_USER_LIBS =    $(LIB_SYSTOOLS) $(LIB_TOOLS) $(LIB_TEXT)
 
 MY_BIN15_LINKLIBS =     User32.lib
 
diff --git a/SelfServiceCommon/Massai/cpp/Systools/src/TestSysProcessTerminate.cpp b/SelfServiceCommon/Massai/cpp/Systools/src/TestSysProcessTerminate.cpp
index 6a7b26c7..219a9d7d 100644
--- a/SelfServiceCommon/Massai/cpp/Systools/src/TestSysProcessTerminate.cpp
+++ b/SelfServiceCommon/Massai/cpp/Systools/src/TestSysProcessTerminate.cpp
@@ -14,7 +14,7 @@ using MASSAI::trace::TraceClient;
 
 int main(int argc, char** argv)
 {
-  TraceClient trc("TestSysProcessTerminate");
+  TraceLocal trc("TestSysProcessTerminate");
   Materna::MSSD::eventSemaphore terminate;
   setShutdownHook(SHUTDOWN_NORMAL, &terminate, "TestSysProcessTerminate", L"cleanup platform", &trc);
   terminate.wait();
-- 
2.41.0.windows.1

